db-developments/devdb.sh

```diff
@@ -42,12 +42,14 @@ function output {
 }
 
 function library_archive {
-    get_version $2
+    local version=$(get_version $2)
+    echo "version of ${2} is ${version}"
     docker run --rm\
         -e AWS_S3_ENDPOINT=$AWS_S3_ENDPOINT\
         -e AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID\
         -e AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY\
-        -e AWS_S3_BUCKET=$AWS_S3_BUCKET\
+        -e AWS_S3_BUCKET=$recipes_bucket\
+        -e CI=true\
```

Comment:
Nice. I'm curious if we can just use the library archive action in CI rather than relying on these calls but that should be out of scope here I think anyways. So that answers my above note on maybe moving whole job to using published image

Reply:
Actually shouldn't this be `CI=$CI`? Could still be run locally

Reply:
Empty string for CI is fine for data library to recognize that it's not being run in CI, so that should be fine